src/flatbuffers/schema/genai_lora.fbs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Generators.lora_parameters;
```

Comment:
May we use Pascal casing for the namespace so it is consistent across the project?

Reply:
clang-format actually specifies Google style which mandates lowercase namespace. It would actually match onnxruntime. So it should be the other way around.